docs/reference/facemesh.md

```diff
@@ -51,7 +51,7 @@ const facemesh = ml5.facemesh(?video, ?options, ?callback);
   detectionConfidence: 0.9, // Threshold for discarding a prediction. Defaults to 0.9.
   maxFaces: 10, // The maximum number of faces detected in the input. Should be set to the minimum number for performance. Defaults to 10.
   iouThreshold: 0.3, // A float representing the threshold for deciding whether boxes overlap too much in non-maximum suppression. Must be between [0, 1]. Defaults to 0.3.
-  scoreThreshold: 0.75, // defaults to 0.75
+  scoreThreshold: 0.75, // A threshold for deciding when to remove boxes based on score in non-maximum suppression. Defaults to 0.75
```

Comment:
Noting here that `score in non-maximum suppression` is not familiar terminology. Looking it up myself I just watched a bit of this video https://www.coursera.org/lecture/convolutional-neural-networks/non-max-suppression-dvrjH. We could refer to a resource and/or simplify the language to something like:

```
// threshold for removing multiple (likely duplicate) detections
```

or
```
// threshold for removing multiple (likely duplicate) detections based on a "non-maximum suppression" algorithm. 
```

Reply:
+1 for  `// threshold for removing multiple (likely duplicate) detections based on a "non-maximum suppression" algorithm. `

Reply:
Sounds good to me!